data/dev/gigadb_user.csv

```diff
@@ -7,3 +7,4 @@ id,email,password,first_name,last_name,affiliation,role,is_activated,newsletter,
 336,test+336@gigasciencejournal.com,5a4f75053077a32e681f81daa8792f95,Hugh,Shanahan,"Department of Computer Science, Royal Holloway, University of London, Egham, TW20 0EX, UK","",f,t,f,,,,,test+336@gigasciencejournal.com,,EBI
 ,admin@gigadb.org,5a4f75053077a32e681f81daa8792f95,Joe,Bloggs,BGI,admin,true,false,true,,,,,admin@gigadb.org,,EBI
 ,user@gigadb.org,5a4f75053077a32e681f81daa8792f95,John,Smith,BGI,user,true,false,true,,,,,user@gigadb.org,,EBI
+345,test+5@gigasciencejournal.com,5a4f75053077a32e681f81daa8792f95,John,Smith,BGI,user,t,f,t,,,,,test+5@gigasciencejournal.com,EBI
```

Comment:
This user record should be moved to before the admin@gigadb.org record. Also fix the email address so it is consistent with the id.